group_2.txt

```diff
@@ -3,4 +3,5 @@ Tri Hoang: [School], [Fun fact]
 Thang Nguyen: [Toronto Metropolitan University],
  [I like snowboarding, badminton and baksketball]
 Quynh Tran: [School], [Fun fact]
+- [DePauw], [Hi]
```

Comment:
Looks like the way you add your information and the way Thang add are not consistent, please reformat